### PR TITLE
util: address few TODO's

### PR DESCRIPTION
**Pull request by Madhu-1**
Addressing a few TODO's in the code base.

### CODE REVIEW

**Comment by Madhu-1** [comment]
Removed last commit as its addressed here https://github.com/ceph/ceph-csi/pull/4227

**Comment by nixpanic** [comment]
Hah, #4227 addresses the 4th commit as well, and cleans up the references even more. Maybe there is no conflict, and both PRs can get merged.

File: internal/util/csiconfig.go [hunk 1]
```diff
@@ -39,11 +39,8 @@ const (
 // ClusterInfo strongly typed JSON spec for the below JSON structure.
 type ClusterInfo struct {
 	// ClusterID is used for unique identification
-	ClusterID string `json:"clusterID"`
-	// RadosNamespace is a rados namespace in the pool
-	RadosNamespace string `json:"radosNamespace"` // For backward compatibility. TODO: Remove this in 3.7.0
-	// Monitors is monitor list for corresponding cluster ID
-	Monitors []string `json:"monitors"`
+	ClusterID string   `json:"clusterID"`
+	Monitors  []string `json:"monitors"`
```
Comment (nixpanic) [inline internal/util/csiconfig.go]: Is there a good reason to remove the comment for `Monitors`?

Comment (Madhu-1) [reply internal/util/csiconfig.go]: its a overside, fixed it now

**Review by nixpanic — APPROVED** [review]

**Review by yati1998 — APPROVED** [review]

**Review by riya-singhal31 — APPROVED** [review]

**Comment by riya-singhal31** [comment]
@Mergifyio queue

**Pull request merged**